src/components/PollCard/PollCard.tsx

```diff
@@ -57,10 +57,21 @@ const PollCard: React.FC<PropTypes> = ({ initialPoll }) => {
   const [poll, setPoll] = useState<Poll>(initialPoll);
   const classes = useStyles();
   const { author, contents: { left, right }, vote } = poll;
+  const { enqueueSnackbar } = useSnackbar();
   const date: string = new Date(poll.createdAt).toLocaleString('default', DATE_FORMAT);
 
+
+  const showSnackBar = (message: string) => {
+    enqueueSnackbar(message, {
+      variant: 'error'
+    });
+  };
+
   const handleVote = (which: Which) => {
-    if (vote) return;
+    if (vote) {
+      showSnackBar('You have already voted');
+      return;
+    }
```

Comment:
I think we don't need this particular function `showSnackBar` - it's only used once and all it does is it adds `variant: 'error'` which is not even mentioned in it's name! Just use: 
```
if (vote) enqueueSnackbar('You have already voted', { variant: 'error' });
```